pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -98,22 +90,6 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 	addFolderMigrations(mg)
 }
 
-func addMigrationLogMigrations(mg *Migrator) {
```

Comment:
^^ this migration is getting moved into the core migration service.  It has to be the first one defined or everything breaks